gke-scripts/td-setup.sh

```diff
@@ -0,0 +1,30 @@
+#! /bin/sh
+
+set -euxo pipefail
```

Comment:
`-o pipefail` is not available in regular sh, it's unique to `bash`.

Reply:
Thanks for the tip. I didn't notice any errors because /bin/sh is symlinked to /bin/bash on Linux (but not on MacOS). I'll change all /bin/sh to /bin/bash

Reply:
done

Reply:
Thanks! It also heavily depends on Linux distro.